init.swift

```diff
@@ -0,0 +1,131 @@
+import Foundation
+
+let baseProjectName = "ios-base"
+var projectName = "RSDemoProject"
+let baseDomain = "com.rootstrap"
+var bundleDomain = baseDomain
+
+let whiteList: [String] = [".DS_Store", "UserInterfaceState.xcuserstate", "init.swift"]
+let fileManager = FileManager.default
+var currentFolder: String {
+  return fileManager.currentDirectoryPath
+}
+
+enum SetupStep: Int {
+  case nameEntry = 1
+  case bundleDomainEntry
+  
+  var question: String {
+    switch self {
+    case .nameEntry: return "Enter name for the project."
+    case .bundleDomainEntry: return "Enter the reversed domain of your organization."
+    }
+  }
+}
+
+// Helper methods
+
+func prompt(message: String) -> String? {
+  print("\n" + message)
+  let answer = readLine()
+  return answer == nil || answer == "" ? nil : answer!
+}
+
+func setup(step: SetupStep, defaultValue: String) -> String {
+  let result = prompt(message: "\(step.rawValue). " + step.question + "(Leave blank for \(defaultValue))")
+  if result == nil {
+    print(defaultValue)
+    return defaultValue
+  }
+  return result!
```

Comment:
a guard may be better to avoid force unwrapping 

```
  guard let result = result else {
     return defaultValue
  }

  return result 
```

Reply:
Totally agree!